application/backend/src/main/java/io/typerefinery/websight/models/components/BaseComponent.java

```diff
@@ -153,7 +163,17 @@ protected void init() {
             componentClasses = Arrays.stream(style.getClasses())
             .collect(Collectors.toCollection(LinkedHashSet::new))
             .toArray(new String[]{});
-        }
+
+            // Width
+            grid.addClasses(gridConfig.get("lgColSize") + getLgColSize());
```

Comment:
can this be done in grid class?

Reply:
now all these things are calculated from the functions which is defined in the DefaultStyledGridComponent